Dockerfile

```diff
@@ -10,15 +10,12 @@ RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
 RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
 RUN apt-get update
 
-WORKDIR /opt
-COPY google-cloud-sdk /opt/google-cloud-sdk
-
-RUN /opt/google-cloud-sdk/bin/gcloud auth list
-
 # Copy the kernel into the container
 WORKDIR /swift-apis
 COPY . .
 
+RUN if test -f google-cloud-sdk; then mv google-cloud-sdk /opt/google-cloud-sdk; opt/google-cloud-sdk/bin/gcloud auth list; fi
```

Comment:
Do you need `opt/google-cloud-sdk/bin/gcloud auth list` also in the command, or can it be skipped?